app/src/main/java/org/mozilla/fenix/share/ShareController.kt

```diff
@@ -101,6 +101,7 @@ class DefaultShareController(
     }
 
     override fun handleShareToApp(app: AppShareOption) {
+        Events.shareToApp.record(Events.ShareToAppExtra(app.packageName))
```

Comment:
Missed this in the first pass, could you add a unit this for this metrics recording to `ShareControllerTest`

Reply:
oh definitely, thanks for catching

Reply:
Added